app/components/modal-delete-integration.js

```diff
@@ -0,0 +1,25 @@
+import ModalComponent from 'ghost-admin/components/modal-base';
+import {alias} from '@ember/object/computed';
+import {inject as service} from '@ember/service';
+import {task} from 'ember-concurrency';
+
+export default ModalComponent.extend({
+    router: service(),
+    notifications: service(),
+    integration: alias('model'),
+    actions: {
```

Comment:
Stylistic thing but we generally keep everything spaced apart in Ghost-Admin. Single-line properties can be grouped such as the service imports, and one-line CPs but everything else has line breaks.